### ISSUE TITLE
Can't read own OTR messages since 1.3.0 

### ISSUE THREAD

**Issue by fauno** [majn/telegram-purple#316]
I just upgraded to 1.3.0 and I can start an OTR conversation with a Telegram contact, but when I write I see the encrypted message printed on the chat window instead of the plain text version.  The contact reports being able to read my messages, but also can't see their own.

**Comment by BenWiederhake** [comment]
What precisely do you mean by "OTR"? Do you mean "native" secret chats in telegram-purple, or do you use telegram-purple together with other plugins, e.g., the OTR plugin?

**Comment by fauno** [comment]
I mean the Off-The-Record plugin, is there anything else going by OTR?

**Comment by majn** [comment]
Just to be sure: The other contact is using pidgin or adium with OTR too, right?

**Comment by fauno** [comment]
yes, both ends are using pidgin with pidgin-otr and telegram-purple
1.3.0

my versions are:

pidgin 2.11.0-1
pidgin-otr 4.0.2-1

-- 
:D

**Comment by majn** [comment]
So, does it work with other protocols?

**Comment by fauno** [comment]
@BenWiederhake hey, sorry I was rude to you, I was stressed at work :(

OTR does work with other protocols, and it was working up to telegram-purple 1.2.6.  The private session is started ok and the messages seem to be sent encrypted, it's just that they're printed as cipher text instead of plain text.

I've noticed regular non-OTR messages take a few seconds to be printed on chat windows, I'm guessing they're only printed when the server receives them?  Maybe this has something to do with printing the cipher text?

**Comment by fauno** [comment]
![probando](https://cloud.githubusercontent.com/assets/468204/19701572/547b2e20-9ad2-11e6-8e18-e1d439a753a3.png)

**Comment by fauno** [comment]
![probando](https://cloud.githubusercontent.com/assets/468204/19701656/aee99586-9ad2-11e6-99a8-6bd20d68b78e.png)

(Removed my phone number...)

**Comment by majn** [comment]
Can you start new OTR chats?

**Comment by steal82** [comment]
same problem here 
Pidgin 2.10.9 (libpurple 2.10.9)
'Off-the-Record' 4.0.0
telegram-purple 1.3.0-1~webupd8~trusty1

I can write otr messages to my contact
my contact can read them
but I see otr text on my side

My contact can write otr messages to me,
I can read them clearly.
but my contact see otr text on his side
